Source/KSCrash/Recording/Tools/KSLogger.h

```diff
@@ -220,7 +220,7 @@ void i_kslog_logCBasic(const char* fmt, ...);
 #define KSLogger_Level_Debug 40
 #define KSLogger_Level_Trace 50
 
-#define NONE  KSLogger_Level_None
+#define KS_NONE  KSLogger_Level_None
 #define ERROR KSLogger_Level_Error
```

Comment:
In my opinion, you should rename other defines too. `ERROR`, `WARN`and `INFO` don't seem to look safe